src/main/java/uk/gov/hmcts/reform/preapi/media/MediaKind.java

```diff
@@ -58,8 +62,62 @@ public MediaKind(
     }
 
     @Override
-    public String playAsset(String assetId) {
-        throw new UnsupportedOperationException();
+    public PlaybackDTO playAsset(String assetName, String userId) {
+        if (getAsset(assetName) == null) {
+            throw new NotFoundException("Asset: " + assetName);
+        }
```

Comment:
The `getAsset` method takes care of throwing the exception so just let it bubble up to the global exception handler.
```suggestion
        getAsset(assetName);
```